i18n/it/cachyos_hello.ftl

```diff
@@ -0,0 +1,66 @@
+# About dialog
+about-dialog-title = CachyOS Hello
+about-dialog-comments = Schermata di benvenuto per CachyOS
+
+# Tweaks page
+tweaks = Modifiche
+fixes = Correzioni
+applications = Applicazioni
+removed-db-lock = Il blocco della base di dati di pacman è stato rimosso!
+lock-doesnt-exist = Il blocco della base di dati di pacman non esiste!
+orphans-not-found = Nessun pacchetto orfano trovato!
+package-not-installed = Il pacchetto '{$package_name}' non è stato installato!
+
+# Dns Connections page
+dns-settings = Impostazioni DNS
+select-connection = Seleziona Connessione:
+select-dns-server = Seleziona server DNS:
+apply = Applica
+reset = Reimposta
+dns-server-changed = Il server DNS è stato cambiato con successo!
+dns-server-failed = Impostazione del server DNS non riuscita!
+dns-server-reset = Il server DNS è stato reimpostato!
+dns-server-reset-failed = Reimpostazione del server DNS non riuscita!
+
+# Tweaks page (tweaks)
+tweak-enabled-title = {$tweak} abilitato
+
+# Tweaks page (fixes)
+remove-lock-title = Rimuovi il blocco della base di dati
+reinstall-title = Reinstalla tutti i pacchetti
+refresh-keyrings-title = Ricarica i keyring
+update-system-title = Aggiorna il sistema
+remove-orphans-title = Rimuovi gli orfani
+clear-pkgcache-title = Pulisci la cache dei pacchetti
+rankmirrors-title = Classifica i mirror
+dnsserver-title = Cambia server DNS
+
+# Main Page (buttons)
+button-about-tooltip = Informazioni
+button-web-resource-tooltip = Risorse in rete
+button-development-label = Sviluppo
+button-software-label = Software
+button-donate-label = Dona
+button-forum-label = Forum
+button-installer-label = Lancia l'installer
+button-involved-label = Partecipa
+button-readme-label = Leggimi
+button-release-info-label = Informazioni di rilascio
```

Comment:
i think it can be more appropriate to use `Note sulla versione`

Reply:
Ok, I've seen both being used more or less, I can pick yours up!